### PR TITLE
upd documentation for new release

### PR DESCRIPTION
**Pull request by jcharlet**

### CODE REVIEW

File: droid-help/src/main/resources/Web pages/FAQ.html [hunk 1]
```diff
@@ -1,446 +1,446 @@
...
+      </li>
+      <li>
+        <a href="FAQ.html#StartFailure">What can I do if DROID fails to start up?</a>
+      </li>
+    </ul>
+    <h3>
+       <a id="JavaVersion" name="JavaVersion">What</a> version of Java do I need?
+    </h3>
+    <p>
+      If you are on windows, a package is provided with Java JDK. To use the version without Java bundled, you will need to have Java 8 to 12 installed on your machine in order to run DROID.
```
Comment (sparkhi) [inline droid-help/src/main/resources/Web pages/FAQ.html]: We bundle the JRE from the JDK right? I think this should read as _a package is provided which also includes JRE with the droid binary_

**Comment by jcharlet** [comment]
do not merge, for some reason the GUI doesn't want to open the github link in another window!....

**Comment by jcharlet** [comment]
travis ran OK on https://travis-ci.org/digital-preservation/droid/builds/644798210?utm_medium=notification&utm_source=github_status

**Comment by jcharlet** [comment]
do not ask me why but droid treats the https links in the help as "show link in web view in DROID", and http links as "open it in browser". Mind blowing.

**Pull request merged**